docs/templates/contributing/04_Writing_dataloader.py.md

```diff
@@ -85,9 +85,9 @@ Here is an example dataloader that gets as input a [fasta](http://genetics.bwh.h
 from __future__ import absolute_import, division, print_function
 import numpy as np
 from pybedtools import BedTool
-from genomelake.extractors import FastaExtractor
 from kipoi.data import Dataset
 from kipoi.metadata import GenomicRanges
+from kipoi.helper import FastaExtractor
 
 class SeqDataset(Dataset):
```

Comment:
Could be replaced by kipoiseq.FastaStringExtractor followed by kipoiseq.transforms.one_hot_dna